geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/iterators/AggregatingScan.scala

```diff
@@ -26,7 +26,7 @@ trait AggregatingScan[T <: AnyRef { def isEmpty: Boolean; def clear(): Unit }] e
 
   protected def manager: GeoMesaIndexManager[_, _, _]
 
-  private var sft: SimpleFeatureType = _
+  var sft: SimpleFeatureType = _
```

Comment:
can we keep this private (since it's a var) and use the one passed into `initResult`